dashboard/config/locales/blocks.en.yml

```diff
@@ -348,9 +374,8 @@ en:
             '"splatter"': Bubbles
             '"swirl"': Hypno
             '"snowflakes"': Snow
-            '"text"': Song Title
             '"spiral"': Spiral
-            "['disco', 'color_cycle', 'rainbow', 'text', 'diamonds','snowflakes','splatter','swirl','spiral'][randomInt(0,8)]": "(Random)"
```

Comment:
Interesting!  Did this cause any errors ('text', ')?

Reply:
What do you mean?